README.md

```diff
@@ -50,6 +50,32 @@ You can also build just one of the binaries in this repo by passing a name to th
 $ make build BINARIES=amtool
 ```
 
+## Docker
+
+<!--- example of complete docker run command to run a container --->
```

Comment:
How about linking to the `Procfile` of this repository, as it gives a user a full setup on their local machine.

Reply:
https://github.com/prometheus/alertmanager/blob/master/Procfile

Reply:
I'm talking about an actual invokation of `docker run ...`, using volumes(`-v`) and port(`-p`) mapping as appropriate, then the command arguments. The `Procfile` contains example invokations of the `alertmanager` binary, but it will look different when using `docker run`, so it can be confusing to people less familiar with docker and unix.